src/test/java/com/sap/cloud/yaas/service_sdk/example/java_jersey_wishlist/api/DefaultWishlistsResourceTest.java

```diff
@@ -0,0 +1,159 @@
+package com.sap.cloud.yaas.service_sdk.example.java_jersey_wishlist.api;
```

Comment:
(Empty) duplicate of [DefaultWishlistsResourceTest.java](https://github.com/SAP/yaas_java_jersey_wishlist/blob/master/src/test/java/com/sap/wishlist/api/generated/DefaultWishlistsResourceTest.java)?